javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -25,11 +25,11 @@ options {
   JAVA_UNICODE_ESCAPE=true;
   COMMON_TOKEN_ACTION=true;
   JDK_VERSION = "1.8";
```

Comment:
Well, looking at the changes in Java 9 it seems that it is a superset of 8, so we can keep a single grammar (much less maintenance work!) - but it should be renamed yes :)